src/tracing/errors.ts

```diff
@@ -96,6 +96,9 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
     if (error.message.match(/WebAssembly.instantiate\(\): Wasm code generation disallowed by embedder/)) {
       return null
     }
+
+    // These are caused by user navigation away from the page before a request has finished.
+    if (error.message.startsWith('AbortError')) return null
```

Comment:
nit: This is a typed error. Can you be more specific and match the type/name?
See https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort#:~:text=Note%3A%20When%20abort()%20is%20called%2C%20the%20fetch()%20promise%20rejects%20with%20an%20Error%20of%20type%20DOMException%2C%20with%20name%20AbortError.

Reply:
it doesn't seem to be typed? https://uniswap-labs.sentry.io/issues/3788567314/?query=is%3Aunresolved+aborterror&referrer=issue-stream&statsPeriod=14d&stream_index=0

Reply:
i'll merge but if you think i'm misunderstanding this lmk and i'll happily make a new pr haha